airbyte-webapp/package.json

```diff
@@ -12,7 +12,8 @@
     "format": "prettier --write 'src/**/*.{ts,tsx}'",
     "storybook": "start-storybook -p 9009 -s public --quiet",
     "lint": "eslint --ext js,ts,tsx src",
-    "license-check": "node ./scripts/license-check.js"
+    "license-check": "node ./scripts/license-check.js",
+    "generate-client": "orval && eslint --fix ./src/core/request/GeneratedApi.ts && prettier ./src/core/request/GeneratedApi.ts --write"
```

Comment:
Since this is an auto-generated file that noone should ever touch by hand (and we later not even want to checkin anymore), I believe we should simply ignore eslint for the whole file (via the generated header), instead of formatting and fixing here. That way (while it's still checked in) it will also stay unaffected by further eslint rule changes.